mobile-app/lib/ui/widgets/drawer_widget/drawer_widget_view.dart

```diff
@@ -13,133 +13,101 @@ class DrawerWidgetView extends StatelessWidget {
       viewModelBuilder: () => DrawerWidgtetViewModel(),
       onModelReady: (model) => model.init(),
       builder: (context, model, child) => Drawer(
-        child: SafeArea(
-          child: SingleChildScrollView(
+        child: Scaffold(
```

Comment:
Can you please update this to the current version of the app

Reply:
i have no write permissions to this repo.

Reply:
i have no permissions to write to this repo